activerecord/test/cases/primary_keys_test.rb

```diff
@@ -351,9 +351,9 @@ class Widget < ActiveRecord::Base
     test "schema dump primary key with bigserial" do
       schema = dump_table_schema "widgets"
       if current_adapter?(:PostgreSQLAdapter)
-        assert_match %r{create_table "widgets", id: :bigserial}, schema
+        assert_match %r{create_table "widgets", id: :bigserial, force: :cascade}, schema
       else
-        assert_match %r{create_table "widgets", id: :bigint}, schema
+        assert_match %r{create_table "widgets", id: :bigint, force: :cascade}, schema
```

Comment:
the primary key column spec will always appear before `force: :cascade` option. Not appear after `force: :cascade`.
https://github.com/rails/rails/blob/4d7e96910f7704e08eb193ddf47323cc71a8ab11/activerecord/lib/active_record/schema_dumper.rb#L121-L136